src/styles/main.scss

```
@@ -0,0 +1,67 @@
+$watch-width: 400px;
+$watch-hight: 400px;
+$center-width: 30px;
+$center-hight: 30px;
+$hand-width: 20px;
+$hand-hight: 180px;
+
+@mixin size {
+  height: $watch-hight;
+  width: $watch-width;
+}
```

Let's rewrite it in the same way, I mentioned before, as well.
```suggestion
@mixin square($size) {
  height: $size;
  width: $size;
}
```